{"name": "Проверка логина курьера с неверными login и password", "status": "passed", "steps": [{"name": "Логин курьера", "status": "passed", "steps": [{"name": "Получение логина курьера", "status": "passed", "start": 1717353241840, "stop": 1717353241840}, {"name": "Получение пароля курьера", "status": "passed", "start": 1717353241840, "stop": 1717353241840}], "parameters": [{"name": "login", "value": "''"}, {"name": "password", "value": "''"}], "start": 1717353241840, "stop": 1717353241972}], "start": 1717353241840, "stop": 1717353241972, "uuid": "b602e62b-2003-4774-a306-493029535aea", "historyId": "b61b713de1b6c23a412a9c4f76acb1bc", "testCaseId": "b61b713de1b6c23a412a9c4f76acb1bc", "fullName": "tests.test_courier.TestLoginCourier#test_login_with_wrong_credentials_failure", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestLoginCourier"}, {"name": "host", "value": "NB-4179"}, {"name": "thread", "value": "12564-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}